semantic/io.cpp

```diff
@@ -343,6 +343,8 @@ void parse(TrafficSign &trafficSign, const Json::Value &value)
     parse(trafficSign.views, Json::check(value, "views", Json::arrayValue));
     parse(trafficSign.size, Json::check(value, "size", Json::arrayValue));
     Json::get(trafficSign.className, value, "className");
+    Json::get(trafficSign.content, value, "content");
```

Comment:
I think that better name of this attribute would be "text".

Reply:
Agree, done.